packer/azure-packer.json.pkr.hcl

```diff
@@ -0,0 +1,89 @@
+variable "subscription_id" {
+  type    = string
+  default = ""
+}
+
+variable "tenant_id" {
+  type    = string
+  default = ""
+}
+
+variable "client_id" {
+  type    = string
+  default = ""
+}
+
+variable "client_secret" {
+  type    = string
+  default = ""
+}
```

Comment:
Currently, authorization works in interactive mode( packer displays you the link you need to open to authorize).

Better we use Azure Active Directory Service Principal ( read more [here](https://www.packer.io/docs/builders/azure#azure-active-directory-service-principal)).
These variables are used for SP authorization, but I failed to run packer using them.